auth0/data_source_helpers.go

```diff
@@ -0,0 +1,73 @@
+package auth0
```

Comment:
This entire file is a direct copy from @yinzara 's #363 , I have made no other changes.